client/src/app/users/user-profile.component.scss

```diff
@@ -0,0 +1,6 @@
+.error-card {
+  background-color: #b00020;
```

Comment:
I don't like having this color hard-coded here. I tried using the color palette info from the themes but couldn't get it to work. This is the hex code from the red provided in the error palette, though.

Reply:
This seems like something @floogulinc might know about. I would love to get a recorded walkthrough about all of the theming stuff so that I could understand that better.

Reply:
Me too! I don't feel like I've ever understood what @floogulinc did there, and I'm not sure the students have been able to use the theming in a very effective way.